chat.js

```diff
@@ -618,6 +653,15 @@ class CommandContext {
 			}
 
 			if (room) {
+				if (room.r9k && !room.isMuted(user)) {
```

Comment:
No it should still add the phrases to the r9k cache. The exception for staff is later in the checkR9k function.